Makefile

```diff
@@ -101,6 +101,7 @@ install_cli_deps: ## Installs `protoc-gen-go`, `mockgen`, 'protoc-go-inject-tag'
 	go install "github.com/golang/mock/mockgen@v1.6.0" && mockgen --version
 	go install "github.com/favadi/protoc-go-inject-tag@latest"
 	go install "github.com/deepmap/oapi-codegen/cmd/oapi-codegen@v1.11.0"
+	curl -fsSL https://raw.githubusercontent.com/tilt-dev/tilt/master/scripts/install.sh | bash
```

Comment:
Also, I guess that `helm` CLI binary is a requirement that should be considered in the `install_cli_deps` as well

Reply:
> this command fails with

Oh no, I was sure tilt install script can be installed on Linux because the same script is offered for both macOS and Linux. https://docs.tilt.dev/install.html

Looking at the script, even on Linux, they check if `brew` CLI is available. That is probably because [Homebrew supports Linux](https://docs.brew.sh/Homebrew-on-Linux). When `brew` cli is available, they try to install `tilt` with it. Do you use `brew` a lot on your Linux WSL2 set up? It is possible your local `brew` set up might be out of date, have you tried to run the commands mentioned to fix this?

It would be easier to install `tilt` with `go install` like we do with other binaries, but unfortunately this is [not supported](https://github.com/tilt-dev/tilt/issues/5334).


> Also, I guess that `helm` CLI binary is a requirement that should be considered in the `install_cli_deps` as well

Thanks for mentioning this, I added a helm install script to that make target. 🫡